_topic_maps/_topic_map.yml

```diff
@@ -3141,10 +3140,35 @@ Topics:
     File: installing-knative-serving
   - Name: Installing Knative Eventing
     File: installing-knative-eventing
-  - Name: Upgrading the OpenShift Serverless Operator
-    File: upgrading-serverless
   - Name: Removing OpenShift Serverless
     File: removing-openshift-serverless
+- Name: Get started
```

Comment:
Get started appears after rel notes and install in the map now, but the "Next steps" in get started are to install the operator and look at the rel notes. Seems like it should come before those two?
![image](https://user-images.githubusercontent.com/61474374/146445462-cc0b0e34-b14d-4d96-9ed6-87a6e4860c77.png)

Reply:
Good catch. I haven't reviewed the content for all the sections yet, I'm just trying to get the buckets in line with the categories in https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=CCS&title=Categories+Shared+Across+Document+Sets so there's a good chance that other sections will need editing too or that this content will move. I'll remove these next steps for now though, thanks 🙂